Source/DafnyCore/Backends/SinglePassCodeGenerator/SinglePassCodeGenerator.cs

```diff
@@ -2749,7 +2750,10 @@ protected class NullClassWriter : IClassWriter {
         Coverage.Instrument(f.Body.tok, $"entry to function {f.FullName}", w);
         Contract.Assert(enclosingFunction == null);
         enclosingFunction = f;
+        CompileReturnBody(f.Body, f.OriginalResultTypeWithRenamings(), w, accVar);
+#if NEW_ATTEMPT
```

Comment:
Is this relevant beyond debugging?

Reply:
Oops. That was leftover from straightening out a merge. Thanks for catching that.